### PR TITLE
Address feedback received on Contribute Code/Docs documentation

### PR DESCRIPTION
**Pull request by satendrapalsingh**
This PR collects and addresses the feedback received so far on the existing [Contribute Code/Docs](https://docs.fragcolor.xyz/contribute/) documentation.

### CODE REVIEW

File: docs/docs/architecture-guides/.pages [hunk 1]
```diff
@@ -0,0 +1,4 @@
+nav:
+  - index.md
+  - chainblocks.md
+  - ...
```
Comment (Kryptos-FR) [inline docs/docs/architecture-guides/.pages:4]: It isn't really necessary: by default `index.md` is always first followed by all pages in alphabetical order. It is mostly useful when the order should be different.

It's not a mistake to have it though.

File: docs/docs/how-to/.pages [hunk 2]
```diff
@@ -0,0 +1,5 @@
+nav:
+  - index.md
+  - chainblocks-primer.md
+  - contributing-turorials.md
+  - ...
```
Comment (Kryptos-FR) [inline docs/docs/how-to/.pages:5]: Same remark.

File: docs/docs/how-to/.pages [hunk 3]
```diff
@@ -0,0 +1,5 @@
+nav:
+  - index.md
+  - chainblocks-primer.md
+  - contributing-turorials.md
```
Comment (Kryptos-FR) [inline docs/docs/how-to/.pages:4]: Typo
```suggestion
  - contributing-tutorials.md
```

File: docs/docs/architecture-guides/.pages (hunk 1 above)
Comment (satendrapalsingh) [reply docs/docs/architecture-guides/.pages:4]: Ok, let's leave it as it for now.

File: docs/docs/how-to/.pages (hunk 2 above)
Comment (satendrapalsingh) [reply docs/docs/how-to/.pages:5]: Noted. Leaving it in for now.

File: docs/docs/how-to/.pages (hunk 3 above)
Comment (satendrapalsingh) [reply docs/docs/how-to/.pages:4]: Fixed

**Review by sinkingsugar — APPROVED** [review]

**Review by Kryptos-FR — APPROVED** [review]

**Pull request merged**